### PR TITLE
coverPrefs

### PR DESCRIPTION
**Pull request by forresto**
crop
filter
overlay

### CODE REVIEW

**Comment by forresto** [comment]
Need anything else for this @jonnor @bergie ?

**Comment by jonnor** [comment]
Tests?

**Comment by forresto** [comment]
Like what?

**Comment by jonnor** [comment]
Like one example of invalid data, and one of valid data in a examples/metadata.yaml file would do the trick. Following the pattern of the other example files.

File: schemata/metadata.yaml [hunk 1]
```diff
@@ -81,4 +81,27 @@ properties:
...
+        name: crop
+        type: boolean
+        description: Allow image cropping
+        example: true
+      overlay:
+        name: overlay
+        type: boolean
+        description: Allow image overlaying
+        example: true
+    crop: true
```
Comment (jonnor) [inline schemata/metadata.yaml]: The values for crop, filter, overlay should not be in schema.

**Comment by forresto** [comment]
examples/metadata.yaml doesn't exist. I added a good example to [examples/contentblock.yml](https://github.com/the-grid/apidocs/blob/0881d860210c81846fb9f56c96321f25c01e8737/examples/contentblock.yml#L42-L45) but don't understand what kind of invalid example would be helpful.

**Comment by forresto** [comment]
OK, added a fail. 

Ready.

**Comment by jonnor** [comment]
Thanks.
In an attempt to clarify: The purpose of _invalid_ type tests is primarily to know what the limitations specified in schema is (correctly) applied. The purpose of _valid_ type tests is primarily to ensure that the data we intend to accept is actually accepted and that we continue to accept it as schema evolves (not breaking compatibility).

**Comment by forresto** [comment]
Yeah. Was heartening to first make the expected fail fail, then pass.

**Comment by jonnor** [comment]
Mo points!!

**Pull request merged**